components/plugin_marketplace/marketplace_list/navigation_row/navigation_row.tsx

```diff
@@ -3,29 +3,29 @@
 
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
-import PropTypes from 'prop-types';
 import {changeOpacity, makeStyleFromTheme} from 'mattermost-redux/utils/theme_utils';
+import {Theme} from 'mattermost-redux/types/preferences';
 
 import NavigationButton from './navigation_button';
 
-export default class NavigationRow extends React.PureComponent {
-    static propTypes = {
-        page: PropTypes.number.isRequired,
-        total: PropTypes.number.isRequired,
-        maximumPerPage: PropTypes.number.isRequired,
-        onNextPageButtonClick: PropTypes.func.isRequired,
-        onPreviousPageButtonClick: PropTypes.func.isRequired,
-        theme: PropTypes.object.isRequired,
-    };
+export type NavigationRowProps = {
+    page: number,
+    total: number,
+    maximumPerPage: number,
+    onNextPageButtonClick: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void,
+    onPreviousPageButtonClick: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void,
+    theme: Theme,
+}
```

Comment:
Nit, types are generally specified with semicolons in the webapp repo

```suggestion
    theme: Theme;
};
```